docs/en/annotations.rst

```diff
@@ -116,39 +116,19 @@ This creates a simple annotation reader with no caching other than in
 memory (in php arrays). Since parsing docblocks can be expensive you
 should cache this process by using a caching reader.
 
-You can use a file caching reader, but please note it is deprecated to
-do so:
+To cache annotations, you can create a ``Doctrine\Common\Annotations\PsrCachedReader``.
+This reader decorates the original reader and stores all annotations in a PSR-6
+cache:
 
 .. code-block:: php
 
-    use Doctrine\Common\Annotations\FileCacheReader;
     use Doctrine\Common\Annotations\AnnotationReader;
+    use Doctrine\Common\Annotations\PsrCachedReader;
+    use Symfony\Component\Cache\Adapter\ApcuAdapter;
 
-    $reader = new FileCacheReader(
+    $reader = new PsrCachedReader(
         new AnnotationReader(),
-        "/path/to/cache",
-        $debug = true
-    );
-
-If you set the ``debug`` flag to ``true`` the cache reader will check
-for changes in the original files, which is very important during
-development. If you don't set it to ``true`` you have to delete the
-directory to clear the cache. This gives faster performance, however
-should only be used in production, because of its inconvenience during
-development.
-
-You can also use one of the ``Doctrine\Common\Cache\Cache`` cache
-implementations to cache the annotations:
-
-.. code-block:: php
-
-    use Doctrine\Common\Annotations\AnnotationReader;
-    use Doctrine\Common\Annotations\CachedReader;
-    use Doctrine\Common\Cache\ApcCache;
-
-    $reader = new CachedReader(
-        new AnnotationReader(),
-        new ApcCache(),
+        new ApcuAdapter(),
```

Comment:
Maybe write this without hardcoding symfony/cache in the readme as any PSR-6 implementation is OK:

```
$cache = ... // instantiate a PSR-6 Cache pool

$reader = new PsrCachedReader(
    new AnnotationReader(),
    $cache,
    $debug = true
);
```

Reply:
Ah, good point. Will update 👍